lib/models/blueprint.js

```diff
@@ -1058,29 +1061,57 @@ Blueprint.prototype.addBowerPackagesToProject = function(packages, installOption
   @return {Promise}
 */
 Blueprint.prototype.addAddonToProject = function(options) {
-  var taskOptions = {};
+  return this.addAddonsToProject({
+    'packages': [options],
+    extraArgs: options.extraArgs || {},
+    blueprintOptions: options.blueprintOptions || {}
+  });
+};
 
-  if (typeof options === 'string') {
-    taskOptions['packages'] = [options];
-  } else {
-    if (!options.name) {
-      throw new SilentError('You must provide a `name` to addAddonToProject');
-    }
+/**
+  Used to add multiple addons to the project's `package.json` and run their
+  `defaultBlueprint` if they provide one.
 
-    if (options.target) {
-      options.name += '@' + options.target;
-    }
+  Generally, this would be done from the `afterInstall` hook, to
+  ensure that a package that is required by a given blueprint is
+  available.
 
-    taskOptions['packages']      = [options.name];
-    taskOptions.extraArgs        = options.extraArgs || [];
-    taskOptions.blueprintOptions = options.blueprintOptions || {};
-  }
+  @method addAddonsToProject
+  @param {Object} options
+  @return {Promise}
+*/
+Blueprint.prototype.addAddonsToProject = function (options) {
+  var taskOptions = {
+    'packages': [],
```

Comment:
Why does `packages` need to be quoted here (might have been that way before, but seems slightly odd)?